helix-term/src/commands.rs

```diff
@@ -2247,53 +2248,51 @@ pub fn jump_backward(cx: &mut Context) {
     };
 }
 
-//
-
-pub fn vsplit(cx: &mut Context) {
-    use helix_view::editor::Action;
+pub fn split(cx: &mut Context, vertical: bool) {
```

Comment:
Why not take in action rather than bool? It's more descriptive.

Reply:
I can see where you're coming from, but I figured `Action::Replace` didn't really make sense, so I'd need to add a panic or something on `split(cx, Action::Replace)` if I did use `Action`.

Reply:
Maybe we can refactor split? `Action::Split(Split)` to have another enum `Split` for this?